packages/smb-onboarder/src/components/ui/negotiators-list/negotiators-list.tsx

```diff
@@ -46,6 +46,8 @@ export const tableDownloadHeaders: DataTableRow[] = [
   { readOnly: true, value: 'eTag (DO NOT EDIT)' },
 ]
 
+const sampleHeaders = ['Username', 'Job Title', 'Email Address', 'Telephone', 'Office', 'Status', 'id', '_eTag']
```

Comment:
I think `Office` column should be `officeId`

Reply:
@ant066 The sample file headers should match with the table sheet headers and downloaded file headers so I think **Office** is fine.